src/components/Chart/index.js

```diff
@@ -49,7 +48,7 @@ export class Chart extends Component {
     renderChart() {
         const { type, labels, plugins, ...conditions } = this.props;
         if (plugins) {
-            ChartJS.plugins.unregister(getGlobalsPlugins(plugins));
+            ChartJS.plugins.clear();
```

Comment:
this must always be done in order to isolate the component so that the global plugins do not affect it.